modules/aws-limits.adoc

```diff
@@ -27,7 +27,7 @@ The following table summarizes the AWS components whose limits can impact your a
 
 These instance type counts are within a new account's default limit. To deploy more worker nodes, deploy large workloads, or use a different instance type, review your account limits to ensure that your cluster can deploy the machines that you need.
 
-In most regions, the bootstrap and worker machines uses an `m4.large` machines and the master machines use `m4.xlarge` instances. In some regions, including all regions that do not support these instance types, `m5.large` and `m5.xlarge` instances are used instead.
+In most regions, the worker machines use an `m6i.large` machines and the bootstrap and master machines uses `m6i.xlarge` instances. In some regions, including all regions that do not support these instance types, `m5.large` and `m5.xlarge` instances are used instead.
```

Comment:
Change to:
In most regions, the worker machines use an `m6i.large` **machine** and the bootstrap and master machines **use** `m6i.xlarge` instances. In some regions, including all regions that do not support these instance types, `m5.large` and `m5.xlarge` instances are used instead.